src/content/docs/en/recipes/i18n.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Add internationalization features 
+description: Use dynamic routing and content collections to add i18n support to your Astro site.
+type: recipe
+i18nReady: false
+---
+import FileTree from '~/components/FileTree.astro'
+import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
+
+Astro doesn't have built-in internationalization (i18n) support, but you can build your own i18n solution. In this recipe, you'll learn how to use content collections and dynamic routing to serve content for different languages.
+
```

Comment:
I'd start off with was sort of URL scheme you're going for (what's the output of following this guide). For example `astro.build/[lang]/blog/foo-bar`